apps/asap-server/test/routes/user.route.test.ts

```diff
@@ -354,6 +354,15 @@ describe('/users/ route', () => {
       expect(userControllerMock.update).toBeCalledWith(userId, { onboarded });
     });
 
+    test('Should not call the controller method if parameters are not valid', async () => {
```

Comment:
i think we should be a little more specific here - i wonder why we even got the 400? is it because orcid is of invalid type or because it should not be in the input at all (i think somewhere above Harlan removed it from the input)

Reply:
We got 400 when passing an orcid field (valid and invalid values). 
I think that's correct because Harlam removed it from the input. 
I'll update it, so it covers both cases 👍